help.txt

```diff
@@ -15,6 +15,8 @@ Available options:
         The amount of requests to make before exiting the benchmark. If set, duration is ignored.
   -S/--socketPath
         A path to a Unix Domain Socket or a Windows Named Pipe. A URL is still required in order to send the correct Host header and path.
+  -W/--useWorkers
```

Comment:
I would call it `-t` and accept an integer (the number of threads)

Reply:
`t` is already being used for timeout and `T` for title.

Reply:
then let's use w but with an integer (number of threads)